data/json/items/armor/holster.json

```diff
@@ -95,11 +101,12 @@
         "max_contains_volume": "1 L",
         "max_contains_weight": "2 kg",
         "max_item_length": "35 cm",
-        "moves": 50
+        "moves": 50,
+        "volume_encumber_modifier": 0.3
       }
     ],
     "use_action": { "type": "holster" },
-    "flags": [ "WATER_FRIENDLY", "BELTED", "OVERSIZE" ],
+    "flags": [ "WATER_FRIENDLY", "BELTED", "OVERSIZE", "PALS_LARGE" ],
```

Comment:
A six-pistol baldric holster would cover almost all if not the entire chest, PALS_LARGE means I can fit twelve on my chest with a standard LBV.